samplescalaproject/src/main/scala/example/Hello.scala

```diff
@@ -0,0 +1,9 @@
+package example
+
+object Hello extends Greeting with App {
+  println("Welcome to 30 days of Scala")
```

Comment:
You will then reference the "greeting" 
```suggestion
  println(greeting)
```

Reply:
ok thanks for the observation. I will the changes and revert back

Reply:
*make